src/plugins/telemetry/public/services/telemetry_sender.ts

```diff
@@ -80,17 +89,18 @@ export class TelemetrySender {
             })
         )
       );
-      this.lastReported = `${Date.now()}`;
-      this.saveToBrowser();
     } catch (err) {
-      // ignore err
-    } finally {
-      this.isSending = false;
+      // ignore err and try again but after a longer wait period.
+      this.retryCount = this.retryCount + 1;
+      if (this.retryCount < 20) {
+        // exponentially backoff the time between subsequent retries to up to 19 attempts, after which we give up until the next report is due
+        window.setTimeout(this.sendUsageData, this.delayRetrySend());
+      }
```

Comment:
NIT: maybe we want to add `else { console.log(something) }` to at least surface the error in the console logs? I feel like it's better than just trapping the error.

Reply:
Done: https://github.com/elastic/kibana/pull/117955/files#diff-5a45ad5f1ca947b4b167609152746b1599cf51aac704cd7b181bb73cec63e208R100-R102